src/components/Reports/Reports.jsx

```diff
@@ -1,17 +1,223 @@
+      dueDateLastWeek: moment().tz('America/Los_Angeles').endOf('week').subtract(1, 'week').toISOString(),
+      dueDateBeforeLast: moment().tz('America/Los_Angeles').endOf('week').subtract(2, 'week').toISOString(),
+      activeTab: '1',
+      errors: {},
+      fetchError: null,
+      loading: true,
+    }
+    this.showProjectTable = this.showProjectTable.bind(this);
+    this.showPeopleTable =this.showPeopleTable.bind(this);
+    this.showTeamsTable=this.showTeamsTable.bind(this)
+    this.setActive=this.setActive.bind(this)
+    this.setInActive=this.setInActive.bind(this)
+    this.setAll=this.setAll.bind(this)
+  }
+
+  componentDidMount() {
+    this.props.fetchAllProjects(); // Fetch to get all projects
+    this.props.getAllUserTeams();
+    this.state = {
+      showProjects: false,
+      showPeople: false,
+      showTeams: false,
+      checkActive: ''
+    }
+    this.props.getAllUserProfile();
+  }
+
+  setActive() {
+    this.setState((state) => {
+      return {
+        checkActive:'true'
+      }
+    });
+  }
+
+  setAll() {
+    this.setState((state) => {
+      return {
+        checkActive:''
+      }
+    });
+  }
+
+  setInActive(){
+
+    this.setState(()=>({
+      checkActive:'false'
+    }))
+  }
+
+  showProjectTable() {
+    this.setState(prevState => ({
+      showProjects: !prevState.showProjects,
+      showPeople: false,
+      showTeams: false,
+      // showTasks: false
+    }))
+  }
+
+  showTeamsTable() {
+    this.setState(prevState => ({
+      showProjects: false,
+      showPeople: false,
+      showTeams: !prevState.showTeams,
+    }))
+  }
+
+  showPeopleTable() {
+    this.setState(prevState => ({
+      showProjects: false,
+      showPeople: !prevState.showPeople,
+      showTeams: false,
+    }))
+      }
+
+
+  showTasksTable(){
+    this.setState(prevState => ({
+      showProjects: false,
+      showPeople: false,
+      showTeams: false,
+    }))
+  }
   render() {
+    let { projects} = this.props.state.allProjects;
+    let { allTeams } = this.props.state.allTeamsData;
+    let { userProfiles } = this.props.state.allUserProfiles;
+
+
+
+    if (this.state.checkActive ==='true'){
+      projects = projects.filter(project => project.isActive ===true);
+      userProfiles =userProfiles.filter(user => user.isActive ===true);
+      allTeams =allTeams.filter(team => team.isActive ===true);
+    }
+    else if (this.state.checkActive ==='false'){
+      projects = projects.filter(project => project.isActive ===false);
+      userProfiles =userProfiles.filter(user => user.isActive ===false);
+      allTeams =allTeams.filter(team => team.isActive ===false);
+    }
+
+
     return (
       <div>
-        <div className="jumbotron">ReportsPage</div>
         <div>
           <nav className="navbar navbar-expand-md navbar-light bg-light mb-3 nav-fill">
             <li className="navbar-brand">Generate Report:</li>
           </nav>
+          <a>Select a Category</a>
```

Comment:
If you are going to have the table centered this should be centered as well on top of it.